### DISCUSSION TITLE
Can you please make the downloaded file draggable when download is done?

### DISCUSSION THREAD

**Discussion by adamlahbib** [subhra74/xdm#697] [Ideas]
When the download is complete window shows up, add the ability to drag the file from that window, keep Open the folder option as well.

**Comment by madgoatz** [comment]
![Internet-Download-Manager-IDM-6 27-Offline-Installer-Download](https://user-images.githubusercontent.com/88365986/139344268-b62697ec-94a8-4a50-808d-5d3b4f60cdaa.png)

You mean like this thing in IDM where u can hold & drag & drop the file to a next folder.?
Also I think your should make feature requests under the issues tab

**Reply by adamlahbib** [reply]
Exactly, ok i ll move it there
